test/nodes/test_translator.py

```diff
@@ -1,60 +1,115 @@
+import pytest
+
+import haystack
 from haystack.schema import Document
+from haystack.nodes import TransformersTranslator
 
-import pytest
 
-EXPECTED_OUTPUT = "Ich lebe in Berlin"
-INPUT = "I live in Berlin"
+ORIGINAL = "TEST QUERY"
```

Comment:
Why is this called so? 🤔

Reply:
Naming is hard :smile: I called it `original` as opposed to `translated`. What could be a better name? "original_text"? "source_text"?

Reply:
I'm gonna go for `ORIGINAL_TEXT` for now, let me know if you have better ideas